src/config/kernel.md

```diff
@@ -71,16 +46,22 @@ virtio-net
 
 There are two different methods to blacklist kernel modules, for the initramfs
 and for the booted system. Some modules are loaded by the initramfs very early
-in the boot process, those have to be blacklisted in the initramfs.
+in the boot process; those have to be blacklisted in the initramfs.
 
-To blacklist modules, create a `.conf` file like `/etc/modprobe.d/radeon.conf`
-with the contents:
+To blacklist modules for the booted system, create a `.conf` file like
```

Comment:
This seems a little unclear?

Reply:
Probably, yeah. I will think about how to explain it better.

Reply:
Just updated most of the text about kernel modules, if you want to take a look.